### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by lndncole**

### CODE REVIEW

File: packages/api/src/router/get.ts
Comment (levibrooke) [inline packages/api/src/router/get.ts:1]: Rename router to profiles.ts

File: packages/api/src/router/post.ts
Comment (levibrooke) [inline packages/api/src/router/post.ts:1]: Move profile related routing functions to the renamed profiles.ts router (currently called get.ts)

File: apps/nextjs/src/pages/sign-in/[[...index]].tsx
Comment (levibrooke) [inline apps/nextjs/src/pages/sign-in/[[...index]].tsx:1]: @levibrooke to look into best way to use dynamic document HEADs for our pages.

File: apps/nextjs/src/pages/index.tsx [hunk 1]
```diff
@@ -1,80 +1,104 @@
...
+          </div>
+          {!!isLoading && (
+            <>
+              <div className="flex h-screen w-screen items-center justify-center">
+                Loading...
+              </div>
+              <div></div>
+            </>
+          )}
+          {!isLoading && !!isDesktop && (
```
Comment (zbellerose) [inline apps/nextjs/src/pages/index.tsx]: I'm not sure we "need" Separate desktop/mobile views like this. The benefit of having tailwind is so you can define  different CSS values depending on the size of the screen. For instance using "xl:w-20" would make the width of a component 5rem only if the size of the screen is above 1280px. 

If you wanted completely different looking views for mobile and desktop, the way you have it now would probably be okay. And obviously, if you believe that the way you have it now is better, please let me know so I can understand.

**Comment by levibrooke** [comment]
Closing this PR after reviewing current state of the app.

**Pull request closed without merging**